packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md

```diff
@@ -182,7 +199,7 @@ import { Component, OnInit } from '@angular/core';
 import { Router, NavigationStart} from '@angular/router';
 
 import { OktaAuthService } from '@okta/okta-angular';
-import * as OktaSignIn from '@okta/okta-signin-widget';
+const OktaSignIn = require('@okta/okta-signin-widget');
```

Comment:
what the issue with using es syntax? the original one should work
https://github.com/okta/samples-js-angular/blob/master/custom-login/src/app/login/login.component.ts#L15

Reply:
With `import * as OktaSignIn from '@okta/okta-signin-widget';` I get error:
```
Error: src/app/login.component.ts:7:29 - error TS7016: Could not find a declaration file for module '@okta/okta-signin-widget'. '//devdocs-angular/okta-app/node_modules/@okta/okta-signin-widget/dist/js/okta-sign-in.entry.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/okta__okta-signin-widget` if it exists or add a new declaration (.d.ts) file containing `declare module '@okta/okta-signin-widget';`
```
Another way to fix this would be putting `//@ts-ignore` before import

Reply:
Changed back to import with `//@ts-ignore`

Reply:
cool! let's use the comment to keep the sample focusing on the implementation logic. We can add notes in the future if there is issue reported with this comment.